src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplateBase.java

```diff
@@ -565,11 +598,12 @@ public CreateContainerCmd fillContainerConfig(CreateContainerCmd containerConfig
             final Long shmSizeInByte = Long.valueOf(shmSize * 1024L * 1024L);
             containerConfig.getHostConfig().withShmSize(shmSizeInByte);
         }
-
+        
```

Comment:
Please avoid making unnecessary whitespace changes - keep the code changes as tight as possible.